packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -320,6 +325,18 @@ export class DetailsRow extends BaseComponent<IDetailsRowProps, IDetailsRowState
     selection.toggleIndexSelected(this.props.itemIndex);
   }
 
+  private _onRootRef(root: FocusZone) {
+    const {
+      refs
+    } = this;
+
+    if (root) {
+      refs.root = ReactDOM.findDOMNode(root) as HTMLElement;
```

Comment:
... is there a good reason for this over using resolveRef? (there might be, not sure.) 

Comments might be good to explain what's up here.